react/Autosuggest/Autosuggest.js

```diff
@@ -122,10 +124,10 @@ export default class Autosuggest extends Component {
     };
 
     const allInputProps = {
```

Comment:
Even though this wasn't introduced in this PR, could you clean this up? Otherwise it looks like an error that we're passing `labelProps` through via `allInputProps` (which is the issue this PR is supposed to fix)